src/components/ArticleListItem/ArticleListItem.test.tsx

```diff
@@ -18,26 +19,63 @@ const testArticle = {
   publishedDate: '2022-05-17T13:44:39.796Z',
 }
 
+const rssTestArticle = {
+  id: 'rssArticle123',
+  preview:
+    'In May 2022, the U.S. Space Force held a Cyber Constructive Service (CCS) Credit Board, in which the',
+  publishedDate: 'Aug 01, 2022',
+  source: 'RSS',
+  sourceLink:
+    'https://www.spaceforce.mil/News/Article/3111951/us-space-force-conducts-innovative-talent-acquisition-process/',
+  sourceName: 'SPACEFORCE.mil',
+  title: 'U.S. Space Force conducts innovative talent acquisition process',
+}
+
 describe('ArticleListItem component', () => {
-  it('renders the article preview', () => {
-    render(<ArticleListItem article={testArticle} />)
+  it('renders the article preview of a cms article', () => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
curious what we're ignoring with this?

Reply:
In the LabelRecord type, there is a field named `type` and it's type (I know...confusing) is defined as `'Source' | 'Audience' | 'Base'`. In the mock article for this file, I gave the `type` field a value of `Source` but it kept giving me an error saying that it doesn't match the type, but it does. Not sure why it wouldn't accept it, so I figured it would be fine to ignore the line since it's a test.

Reply:
It looks like if you add the type `ArticleListItemRecord` when defining the const `cmsTestArticle` it will recognize the SearchResultType correct.

```
const cmsTestArticle: ArticleListItemRecord = {
  id: 'testArticleId123',
  labels: [{ id: 'labelId', name: 'A Label', type: 'Source' }],
  ...
}
```

Reply:
You are correct. Fixed it!